webapp/sdf/.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+}
```

Comment:
Just clean code improvements.
```suggestion
}